api/controllers/v0.3/endpoints.js

```diff
@@ -336,6 +337,7 @@ async function createPatron(req, res) {
     address: address, // created above
     workAddress: workAddress,
     username: req.body.username, // from req
+    usernameHasBeenValidated,
```

Comment:
Why doesn't this need a key as well as a value?  Also, I'm not sure you need a separate variable for `usernameHasBeenValidated`; it's a little confusing to read, given that almost everything else in here is just `req.body`.whatever.

Reply:
Yeah, I can clean that up.